app/controllers/pages_controller.rb

```diff
@@ -0,0 +1,6 @@
+class PagesController < ApplicationController
+  def about
+  end
+  def faq
```

Comment:
It's a good practice to leave an empty line after `end` when declaring a new method.